### PR TITLE
WQAB SM building array upgrade Cusp-Mes to EQR120s

### PR DESCRIPTION
**Pull request by Thomas-Benson**
- Upgrading 16 Cusp-Me recorders to EQR120s
- Using EQR120 firmware version 2.3.2
- Updating stream codes from XYZ to 12Z
- All new recorders now have consistent orientation of 10 degrees

### CODE REVIEW

**Review by mnaguit — CHANGES_REQUESTED** [review]
Lots of work here, great job @Thomas-Benson , thanks! 👍 
To check: for `LC 2G`, the serial number from the main ticket is `46526`

File: assets/recorders.csv [hunk 1]
```diff
@@ -611,6 +611,7 @@ Canterbury Seismic Instruments,CUSPM,11068,13654,
 Canterbury Seismic Instruments,CUSPM,11069,13655,
 Canterbury Seismic Instruments,CUSPM,13032,13974,
 Canterbury Seismic Instruments,CUSPM,13041,13004,
+Canterbury Seismic Instruments,EQR120,46426,,
```
Comment (mnaguit) [inline assets/recorders.csv:614]: no need for this entry?

File: install/recorders.csv [hunk 2]
```diff
@@ -652,7 +652,9 @@ Canterbury Seismic Instruments,CUSPM,CUSPM,11068,CPLB,2E,37,,0,-23.12,2009-04-15
 Canterbury Seismic Instruments,CUSPM,CUSPM,11069,CPLB,2F,37,,0,-23.5,2009-04-15T00:00:03Z,2019-01-19T21:00:00Z
 Canterbury Seismic Instruments,CUSPM,CUSPM,13032,CPXB,27,302,,0,-24.5,2010-05-01T00:00:02Z,2020-12-14T19:55:00Z
 Canterbury Seismic Instruments,CUSPM,CUSPM,13041,AVAB,28,240,,0,1.228,2007-11-05T00:00:00Z,9999-01-01T00:00:00Z
+Canterbury Seismic Instruments,EQR120,EQR120,46426,WQAB,2G,10,,0,0,2024-05-22T02:22:00Z,9999-01-01T00:00:00Z
```
Comment (mnaguit) [inline install/recorders.csv:655]: serial number is `46526`

**Comment by Thomas-Benson** [comment]
Fixed mistake in serial numbers @mnaguit

**Review by mnaguit — APPROVED** [review]

**Review by CBurton90 — APPROVED** [review]

**Comment by Thomas-Benson** [comment]
Delta merged

**Pull request merged**